server/src/services/sqlIngest.py

```diff
@@ -242,8 +232,9 @@ def fix_nan_vals(resultDict):
 
 if __name__ == "__main__":
     '''Class DataHandler workflow from initial load to SQL population'''
-    loader = DataHandler()
-    loader.loadConfig(configFilePath='../settings.cfg')
+    config = ConfigParser()
+    config.read('../settings.cfg')
+    loader = DataHandler(config)
     loader.fetchSocrataFull()
```

Comment:
@sellnat77  I think this config rejiggering will break the DataCleaner object in dataCleaner.py since it inherits from the DataHandler class. Is the idea just that we don't want to specify the config file location on each load? I see that this cuts out a method, but this might lead to more overhead in reuse of this code down the road.

Reply:
Yeah the load config method was a super of the class initialization and there was a property that was created in load config but not in the class initialization so maintaining both will lead to headache
We can move the config setup to a separate method and have the class initialization call that method

Reply:
I just want to keep things as dry as we can

Reply:
Gotcha. I think we can also just require the config spec on object initialization and call self.loadConfig to assign it to the object. Forward declaration works in class definitions.